addons/website_slides/models/slide_channel.py

```diff
@@ -541,7 +541,8 @@ def _filter_add_members(self, target_partners, **member_values):
                 on_invite.check_access_rights('write')
                 on_invite.check_access_rule('write')
             except:
-                pass
+                if self._context.get('raise_on_not_allowed_to_add_members'):
+                    raise AccessError(_('You are not allowed to add members to this course. Please contact the administrator.'))
```

Comment:
Maybe "the administrator" is a bit much, you could just ask to contact the course responsible?

Reply:
@awa-odoo I changed the error message with `the course responsible` but do we agree it still a bit weird to have this message while no responsible is set on the course...?

Reply:
Mmmh this can happen when there is a responsible that is not "you" right?

Maybe we could go with "the course responsible or an administrator" for safe measures :slightly_smiling_face:

Reply:
> Mmmh this can happen when there is a responsible that is not "you" right?

I'm more talking of the case when there isn't a responsible set on the course, but still cannot be edited by an officer.

> Maybe we could go with "th e course responsible or an administrator" for safe measures slightly_smiling_face.

Alright, will do :+1: